pkg/model/serializer.go

```diff
@@ -693,8 +693,28 @@ func (d *Manifest) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+func (d *DeployCommand) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
when marshalling, if command and name are the sample, we should marshall the simple syntax. this is important because it's the most common case and we are marshalling the manifest for the UI. also, add tests for it